pkg/cmd/experimental/syncgroups/rfc2307/ldapinterface.go

```diff
@@ -62,6 +63,13 @@ type LDAPInterface struct {
 var _ interfaces.LDAPMemberExtractor = &LDAPInterface{}
 var _ interfaces.LDAPGroupGetter = &LDAPInterface{}
 var _ interfaces.LDAPGroupLister = &LDAPInterface{}
+var _ interfaces.LDAPMemberExtractor = &LDAPInterface{}
+var _ interfaces.LDAPGroupGetter = &LDAPInterface{}
+var _ interfaces.LDAPGroupLister = &LDAPInterface{}
+
+func (e *LDAPInterface) String() string {
```

Comment:
On further reflection, this probably isn't worth it.  If you want %#v, specify it.